docs/meeting-notes.rst

```diff
@@ -29,9 +29,9 @@ To adress different time zones among the globe continental meetings have been or
 
    Gitter: `Pangeo Europe <https://gitter.im/pangeo-data/Europe>`__ 
 
-   - Cofee table; an uninformal meeting with open discussions:   
+   - Pangeo European Coffee ; an uninformal meeting with open discussions:   
       * Schedule: Every week on Tusday at 10a CET
-      * Conferencing: Conferencing: `whereby.com/pangeo-europe <https://whereby.com/pangeo-europe>`_
+      * Conferencing: Conferencing: `meet.jit.si/pangeo-europe <https://meet.jit.si/pangeo-europe>`_
```

Comment:
Tuesday instead of Tusday

Reply:
I would also put 10am instead of 10a (I am not sure but I think it is the "correct" way to specify times).